src/tests/ftest/server/metadata.py

```diff
@@ -310,20 +310,23 @@ def test_metadata_addremove(self):
         containers_created = []
         for loop in range(10):
             self.log.info("Container Create Iteration %d / 9", loop)
+            # The test will encounter ENOSPACE (-1007) while creating
+            # containers.
             if not self.create_all_containers():
                 self.log.error("Errors during create iteration %d/9", loop)
-                test_failed = True
 
             containers_created.append(len(self.container))
-
+            # We should make sure containers which are created should
+            # be destroyed without issues. Here we have to check for
+            # any container destroy errors.
             self.log.info("Container Remove Iteration %d / 9", loop)
             if not self.destroy_all_containers():
                 self.log.error("Errors during remove iteration %d/9", loop)
                 test_failed = True
-
             # Calculate the mean container count
             mean_cont_cnt = mean_cont_cnt + containers_created[loop]
-            mean_cont_cnt = int(mean_cont_cnt / (loop + 1))
+            if loop > 0:
+                mean_cont_cnt = int(mean_cont_cnt / 2)
```

Comment:
This calculation seems incorrect. For example, consider:
```
mean_cont_cnt = 0

containers_created.append(2)
mean_cont_cnt = 0 + 2 = 2

containers_created.append(10)
mean_cont_cnt = 2 + 10 = 12
mean_cont_cnt = int(12 / 2) = 6

containers_created.append(2)
mean_cont_cnt = 6 + 2 = 8
mean_cont_cnt = = int(8 / 2) = 4

containers_created.append(2)
mean_cont_cnt = 4 + 2 = 6
mean_cont_cnt = int(6 / 2) = 3
```
But the mean of `[2, 10, 2, 2] = 4`, not `3`

Why not just do this once at the end?
`mean_cont_cnt = sum(containers_created) / len(containers_created)`

Reply:
Good find. Let me check...